### PR TITLE
A user can see the lists along with their todo items

### PR DESCRIPTION
**Pull request by VolodymyrTymtsiv**
http://10.0.2.113:8080/

### CODE REVIEW

**Comment by sergeyradkevich** [comment]
@VolodymyrTymtsiv @ShAndriy Rename given pull request into the name of the user story please.

**Comment by sergeyradkevich** [comment]
As agreed, let's improve the layout of the main page.

File: src/main/java/ua/realization/CreatorContentTask.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+@Service
+public class CreatorContentTask implements Content {
+
+    @Override
+    public List<ViewListItems> getContent() {
+        List<ViewListItems> listTask = new ArrayList<>();
+
+        List<String> item = new ArrayList<>();
+        String name = "WatchTV";
```
Comment (sergeyradkevich) [inline src/main/java/ua/realization/CreatorContentTask.java]: As it is a collection let's rename into plural form - `items`.

**Comment by sergeyradkevich** [comment]
@VolodymyrTymtsiv and @ShAndriy will proceed on Monday.

**Comment by VolodymyrTymtsiv** [comment]
@sergeyradkevich can view updates.

**Comment by sergeyradkevich** [comment]
Hey guys, can you please prepare a running app that corresponds to given changes of the pull request so I could verify (test) it? In case 8080 port is busy with development needs at http://10.0.2.113, please set up a separate version of the app using another port, for example 3000.

Provide the url for testing here please.

File: src/main/java/TaDa/Property/SiteProperty.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package TaDa.Property;
```
Comment (sergeyradkevich) [inline src/main/java/TaDa/Property/SiteProperty.java]: Use a lower case for package naming please: `tada.property`.

File: src/main/java/TaDa/Property/SiteProperty.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+public final class SiteProperty {
+
+    private String headerName = "TaDa";
```
Comment (sergeyradkevich) [inline src/main/java/TaDa/Property/SiteProperty.java]: I expected some properties/settings file where all static texts would be. But, okay, for now let's leave it as it was. So move the text back please and delete given source file. Later we will se what texts we could move out of the view into properties.

**Comment by VolodymyrTymtsiv** [comment]
On http://10.0.2.113:8080/  all good...can try...If not write me and i set port 3000...

File: src/main/java/TaDa/abstraction/Content.java [hunk 4]
```diff
@@ -0,0 +1,9 @@
+package TaDa.abstraction;
+
+import TaDa.viewBeans.ViewListItems;
+
+import java.util.List;
+
+public interface Content {
+    List<ViewListItems> getContent();
+}
```
Comment (sergeyradkevich) [inline src/main/java/TaDa/abstraction/Content.java]: This is a good point to have an interface 👍 But let's rename it into, for example, `ListItems`.

File: src/main/java/TaDa/controllers/ToDoListController.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import TaDa.Property.SiteProperty;
+
+@Controller
+public class ToDoListController {
+
+    @Autowired
+    private Content service;
```
Comment (sergeyradkevich) [inline src/main/java/TaDa/controllers/ToDoListController.java]: Rename `service` into `useCase` please (will clarify in details soon).

File: src/main/java/TaDa/realization/CreatorContentTask.java [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+import TaDa.abstraction.Content;
+import TaDa.viewBeans.ViewListItems;
+import org.springframework.stereotype.Service;
+import TaDa.viewBeans.ToDoItem;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class CreatorContentTask implements Content {
```
Comment (sergeyradkevich) [inline src/main/java/TaDa/realization/CreatorContentTask.java]: Rename into `ListItemsUseCase` please.

File: src/main/java/TaDa/realization/CreatorContentTask.java [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+
+    @Override
+    public List<ViewListItems> getContent() {
+        List<ViewListItems> listTask = new ArrayList<>();
```
Comment (sergeyradkevich) [inline src/main/java/TaDa/realization/CreatorContentTask.java]: Rename into `getListItems` please.

**Comment by sergeyradkevich** [comment]
Reviewed, take a look please and update.

**Comment by sergeyradkevich** [comment]
@VolodymyrTymtsiv I do not think that that version corresponds to the current changes of given pull request, as there are dates present there :)

**Comment by sergeyradkevich** [comment]
@VolodymyrTymtsiv but it looks fine, btw. Verified!

**Comment by VolodymyrTymtsiv** [comment]
@VolodymyrTymtsiv I do not think that that version corresponds to the current changes of given pull request, as there are dates present there  ----- I don't understand what you mean..

**Comment by VolodymyrTymtsiv** [comment]
We refactor code. Please see whether all good and marge...

**Comment by sergeyradkevich** [comment]
Congrats! 🎉

**Pull request merged**